realm/realm-library/src/objectServer/java/io/realm/mongodb/auth/EmailPasswordAuth.java

```diff
@@ -145,6 +146,19 @@ public void resendConfirmationEmail(String email) throws AppException {
         ResultHandler.handleResult(null, error);
     }
 
+    /**
+     * Retries the custom confirmation function on a user for a given email.
+     *
+     * @param email the email of the user.
+     * @throws AppException if the server failed to confirm the user.
+     */
+    public void retryCustomConfirmation(String email) throws AppException {
+        Util.checkEmpty(email, "email");
```

Comment:
Don't we usually include a test for these validations?

Reply:
Yes, I missed written such test